README.md

```diff
@@ -1,4 +1,8 @@
-# Asgardeo .NET OIDC SDK
+# [DEPRECATED] Asgardeo .NET OIDC SDK
+
+## :warning: Warning!
+### .NET SDK is no longer encoraged and enriched by Asgardeo and may not work with the latest versions.
```

Comment:
The term encoraged should be changed as encouraged

Reply:
Fixed with b28ef716b6ea88ba56526fab61a58d52b92e2de6